#! /bin/sh /usr/share/dpatch/dpatch-run
## manpages_section.dpatch by Samuel Mimram <smimram@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Correct section (3) for manpages.

@DPATCH@
diff -urNad ocaml-3.09.1~/ocamldoc/Makefile ocaml-3.09.1/ocamldoc/Makefile
--- ocaml-3.09.1~/ocamldoc/Makefile	2005-11-10 15:44:36.000000000 +0100
+++ ocaml-3.09.1/ocamldoc/Makefile	2006-01-04 21:14:39.000000000 +0100
@@ -313,7 +313,7 @@
 
 stdlib_man/Pervasives.3o: $(STDLIB_MLIS)
 	$(MKDIR) stdlib_man
-	$(OCAMLDOC_RUN) -man -d stdlib_man $(INCLUDES) \
+	$(OCAMLDOC_RUN) -man -man-section 3 -d stdlib_man $(INCLUDES) \
 	-t "OCaml library" -man-mini \
 	$(STDLIB_MLIS)
 
